### ISSUE TITLE
Possible bug in v4 documentation

### ISSUE THREAD

**Issue by lukasoppermann** [amzn/style-dictionary#1336]
Hey, so I was running into an issue while overwriting `color/hex`.

I followed this docs https://styledictionary.com/reference/hooks/transforms/#defining-custom-transforms and ran `new StyleDictionary()` afterwards.

I now figured out that the transform was overwritten again with the default. It seems I need to run `new StyleDictionary()` first and register the transform afterwards if it has the same name as an existing one. Is this correct?

If this is the expected behavior it would be great to add a yellow warning block in the docs to explain this behavior.

If I am missing something, feel free to correct me. 😅

**Comment by jorenbroekema** [comment]
It depends on whether you're doing this:

```js
import StyleDictionary from 'style-dictionary';

StyleDictionary.registerTransform();
```
or
```js
import StyleDictionary from 'style-dictionary';

const sd = new StyleDictionary();
sd.registerTransform();
```

The first example registers/overwrites the transform that's set on the class level and the latter does it on the instance level.

If you register/overwrite it on the class level, each subsequent created instance AND each prior created instance will both have this hook registered/overwritten.

**Comment by lukasoppermann** [comment]
> The first example registers/overwrites the transform that's set on the class level and the latter does it on the instance level.

I am not sure this is true. I would expect that if it was overwritten, that if I do this `const sd = new StyleDictionary();` I get the _new_ function. But I am getting the original `color/hex` function that was shipped with StyleDictionary. 

Or am I misunderstanding you?

**Comment by jorenbroekema** [comment]
I'm trying to confirm what I'm saying and noticing that we might have found a bug here :D
